Assets/LeapMotion/Modules/GraphicRenderer/Scripts/Graphics/ProceduralMeshGraphics/LeapSlicedGraphic.cs

```diff
@@ -0,0 +1,389 @@
+﻿/******************************************************************************
+ * Copyright (C) Leap Motion, Inc. 2011-2017.                                 *
+ * Leap Motion proprietary and  confidential.                                 *
+ *                                                                            *
+ * Use subject to the terms of the Leap Motion SDK Agreement available at     *
+ * https://developer.leapmotion.com/sdk_agreement, or another agreement       *
+ * between Leap Motion and you, your company or other organization.           *
+ ******************************************************************************/
+
+using Leap.Unity.Attributes;
+using Leap.Unity.Query;
+using UnityEngine;
+using UnityEngine.Rendering;
+
+namespace Leap.Unity.GraphicalRenderer {
+
+  /// <summary>
+  /// The base class for LeapPanelGraphic, LeapBoxGraphic, and similar generators.
+  /// </summary>
+  /// <details>
+  /// This base class represents shared code when constructing graphics that:
+  /// - Are generally rectangular, supporting dynamic tesselation along the X and Y axes
+  /// - Can be intuitively controlled by an associated RectTransform
+  /// - Support nine-slicing when using a Sprite as a source for Texture data
+  /// </details>
+  [DisallowMultipleComponent]
+  public abstract class LeapSlicedGraphic : LeapMeshGraphicBase {
+
+    #region Constants
+
+    public const int MAX_VERTS_PER_AXIS = 128;
```

Comment:
This isn't the case in general, since for boxes your per-axis limit will be lower due to the higher dimensions.